src/app/components/algorithms/implementation-view/implementation-publications-list/implementation-publications-list.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { AlgorithmService } from 'api/services/algorithm.service';
+import { PublicationService } from 'api/services/publication.service';
+import { ActivatedRoute, Router } from '@angular/router';
+import { EntityModelPublicationDto } from 'api/models/entity-model-publication-dto';
+import { EntityModelImplementationDto } from 'api/models/entity-model-implementation-dto';
+import { PublicationDto } from 'api/models/publication-dto';
+import { GenericDataService } from '../../../../util/generic-data.service';
+
+@Component({
+  selector: 'app-implementation-publications-list',
+  templateUrl: './implementation-publications-list.component.html',
+  styleUrls: ['./implementation-publications-list.component.scss'],
+})
+export class ImplementationPublicationsListComponent implements OnInit {
+  @Input() implementation: EntityModelImplementationDto;
+  linkedPublications: EntityModelPublicationDto[] = [];
+  tableColumns = ['Title', 'URL', 'DOI', 'Authors'];
+  variableNames = ['title', 'url', 'doi', 'authors'];
+  linkObject: any = {
```

Comment:
are you required to add any here?

TypeScript should be able to infer the object type here. I'm not sure whether our eslint rules allow that though.